justfile

```diff
@@ -36,3 +36,15 @@ _py-install:
 # Install dependencies for JavaScript
 _js-install:
     cd automations/js && pnpm install
```

Comment:
@AetherUnbound does removing `cd automations/js` from this line fix the error you saw?

Reply:
It does!

Reply:
I can't reproduce this locally, no matter what I tried.